docs/user/metrics/string.md

```diff
@@ -43,6 +45,27 @@ assertTrue(SearchDefault.name.testHasValue())
 assertEquals("wikipedia", SearchDefault.name.testGetValue())
 ```
 
+### Swift
+
+```Swift
+// Record a value into the metric.
+SearchDefault.name.set("duck duck go")
```

Comment:
Ah, so we're not generating the metrics in a Glean namespace? The reason why we had one was to prevent naming collision in the code.

Reply:
There's no namespaces in Swift :(

Reply:
I mean ... we're (mis)using empty enums as namespaces, but you then can't do "import GleanMetrics" to use just `SearchDefault.name`

Reply:
Mh, I see. I would see, for example, `SearchDefault` easily clashing with something else in a browser application :( Maybe we could add a `Glean` prefix to categories? @mdboom , @travis79  thoughts?

Reply:
That's independent of the string metric though, I'll open a bug for discussion as a follow-up

Reply:
Yes, good point. Thanks!

Reply:
Yeah, I like the idea of a `Glean` prefix (given the constraints of the language).

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1582133